app/src/main/res/drawable/ic_app.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you post the screenshot of the old and the new style in the description?

Reply:
![Screenshot_20230518_123254](https://github.com/tomtom-international/tomtom-navigation-android-examples/assets/111105981/caef32fd-f475-4d53-ae30-65197c9f0c00)
![Screenshot_20230518_123435](https://github.com/tomtom-international/tomtom-navigation-android-examples/assets/111105981/abf5c2b0-04ed-4eb1-8536-f20f48482a30)

Reply:
For me it looks like the text is moved to the bottom of the icons, is it me or it was moved?

Reply:
@LukaszWitkowski-tomtom the text isn't as well vertically aligned with the combination of the new icons and font family. I looked into this but as a beginner Android developer I wasn't able to achieve a better alignment. Is this something you could look at?

Reply:
Done: 
![Screenshot_20230522_111330_TomTom SDK Examples](https://github.com/tomtom-international/tomtom-navigation-android-examples/assets/88668840/1337f30c-8720-413c-8dbf-da922f4fdcff)